css/signForm/signForm.css

```
@@ -122,7 +122,13 @@
 }
 
 /* js error event */
-.password-error {
+.errorMsg_style {

```

줄임말없이 쓰는 것이 컨벤션인 회사도 많다고 알고있습니다.
```suggestion
.errorMessage_style
```